ansible/ansible-inventory

```diff
@@ -0,0 +1,2 @@
+[webservers]
+X.X.X.X
```

Comment:
This is wise, but minus points for making me search my emails to test! 😆

Reply:
Sorry about that